Apps/PackageTest/0.63.1/package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
-    "@babylonjs/core": "5.6.1",
-    "@babylonjs/react-native": "^0.4.0-alpha.47",
+    "@babylonjs/core": "5.35.1",
+    "@babylonjs/react-native": "^1.4.1",
```

Comment:
I'm guessing this is broken since it doesn't even reference any of the BRN binary packages. But yea removing it or fixing it in a separate PR makes sense.

Reply:
We can do it in a separate PR.